app/templates/course/stage/instructions.hbs

```diff
@@ -91,6 +91,33 @@
 
     <CoursePage::CourseStageStep::YourTaskCard @repository={{@model.activeRepository}} @courseStage={{@model.courseStage}} />
 
+    {{! TODO: Remove this temporary card  }}
+    {{#if (and @model.courseStage.isSecond (eq @model.courseStage.course.slug "shell"))}}
```

Comment:
@andy1li thoughts on keeping this a bit higher up? Worried that not enough people will see this since it falls below the your task card.